### ISSUE TITLE
有个问题：为啥网络的输出的两个数都趋于0？

### ISSUE THREAD

**Issue by guanyonglai** [yuanlairuci110/pytorch-best-practice-master#1]
我看了下，这个和pytorch的loss函数设置有关，对于CrossEntropyLoss，即使我的输出和label一致，[1,0,0]对0，可是输出的loss还是高达0.5514，从而导致了输出一直在0附近徘徊，而loss一直在0.65附近，请问博主怎么解决这个问题呢？

entroy=nn.CrossEntropyLoss()
output=torch.Tensor([[1, 0,0]])
target = torch.tensor([0])
loss = entroy(output, target)
print(loss)
输出loss：tensor(0.5514)
